config/routes.rb

```diff
@@ -1,6 +1,22 @@
 Rails.application.routes.draw do
-  root to: "wishlists#index"
   resources :wishlists, only: [:index, :new, :create]
-  devise_for :users
+  devise_for :users, skip: [:sessions, :registrations]
```

Comment:
If you skip those functionalities of Devise, there aren't any routes generated left by this declaration.